assets/app/scripts/services/nameGenerator.js

```diff
@@ -14,11 +14,12 @@ angular.module("openshiftConsole")
        */
       suggestFromSourceUrl: function(sourceUrl){
         var projectName = sourceUrl.substr(sourceUrl.lastIndexOf("/")+1, sourceUrl.length);
-        var index = projectName.lastIndexOf(".");
+        var index = projectName.indexOf(".");
```

Comment:
this was actually a bug since the app name cant have any '.' characters, was an issue when I tested with ref that had a '.' in it